guide/miscellaneous/useful-packages.md

```diff
@@ -140,55 +140,119 @@ console.log(chalk.green.bgBrightBlack.underline('This is so pretty.'));
-The `format` option tells the logger which format to use for the messages; by default, it outputs JSON objects.
-While useful, JSON is not very readable, so we define a custom format that displays the log level in all caps alongside the message.
-If you wanted to, you could also use the chalk module to make the logger's format a bit prettier by applying colors, etc.
+Pino logs in a json format, so other programs and services like log aggrregators can easily parse and work with the output. This is very useful for production systems, but quite tedious to read during development. This is why you installed `pino-pretty` earlier. Instead of formatting the log output itself the developers recommended that you [pipe](https://en.wikipedia.org/wiki/Pipeline_(Unix)) the log output to other services instead. `pino-pretty` is a formatter you can use for easy-to-read logs during development.
+
+We recommend you set `pino-pretty` up in a package script in your `package.json` file, rather than typing the pipeline out every time. Please read our [guide section on package scripts](/improving-dev-environment/package-json-scripts), if you are not sure what we're talking about here.
+
+```json {10}
+{
+	"name": "my-bot",
+	"version": "1.0.0",
+	"description": "A Discord bot!",
+	"main": "index.js",
+	"scripts": {
+		"test": "echo \"Error: no test specified\" && exit 1",
+		"start": "node .",
+		"lint": "eslint .",
+		"dev": "node . | pino-pretty -i pid,hostname -t yyyy-mm-dd HH:MM:ss"
+	},
+	"keywords": [],
+	"author": "",
+	"license": "ISC"
+}
+```
+
+:::warning
+If you are using powershell, you have to use a package script for `pino-pretty`. Powershell handles pipelines in a way that prevents logging. The cmd commandline interface is not affected.
+:::
+
+In above example, you can see how to modify the output generated by `pino-pretty` with further arguments. `-i pid,hostname` hides these two elements from logged lines and `-t yyyy-mm-dd HH:MM:ss` formats the timestamp into an easy to use format. Try out what works for you! The official [pino-pretty documentation](https://github.com/pinojs/pino-pretty) explains all possible arguments.
+
+To start your bot with prettified input you run the `dev` script via your package manger of choice:
+
+:::: code-group
+::: code-group-item npm
+```sh:no-line-numbers
+npm run dev
+```
+:::
+::: code-group-item yarn
+```sh:no-line-numbers
+yarn run dev
+```
+:::
+::: code-group-item pnpm
+```sh:no-line-numbers
+pnpm run dev
+```
+:::
+::::
 
-![winston example](./images/winston.png)
+Pino is very flexible, supports custom log levels, worker threads and many many more features. Please check out the [official documentation](https://getpino.io) if you want to up your pino game! Below we show an alternative for a production setup. Using this code, you will be logging the raw json objects into a file, instead of printing to your console: 
```

Comment:
```suggestion
Pino is very flexible, supports custom log levels, worker threads and many more features. Please check out the [official documentation](https://getpino.io) if you want to up your pino game! Below we show an alternative for a production setup. Using this code, you will be logging the raw json objects into a file, instead of printing to your console: 
```